egs/sre16/v1/local/nnet3/xvector/prepare_feats_for_egs.sh

```diff
@@ -41,30 +42,48 @@ done
 # Set various variables.
 mkdir -p $dir/log
 mkdir -p $data_out
-featdir=${PWD}/$dir
+featdir=`readlink -f $dir`
```

Comment:
This command is not portable to BSD.  Use utils/make_absolute.sh instead of readlink of.
Also we prefer $() to backticks.